funnel/templates/comments.html.jinja2

```diff
@@ -16,13 +17,14 @@
           <div class="comment--header__details">
             {% if comment.state.DELETED -%}
               <span class="commenter">[{% trans %}deleted{% endtrans %}]</span>
+            {% elif comment.state.SPAM -%}
+              <span class="commenter">[{% trans %}removed{% endtrans %}]</span>
             {%- else -%}
               <span class="commenter"><strong>{{ comment.user.pickername }}</strong></span>
-              {%- if comment.user == document.user %}
-                <span class="badge">Proposer</span>
-              {% endif %}
-              {%- if 'crew' in project.roles_for(comment.user) %}
-                <span class="badge">Crew</span>
+              {%- if comment.badges %}
+                {% for badge in comment.badges %}
+                  <span class="badge">{{ badge }}</span>
+                {% endfor %}
               {% endif %}
```

Comment:
Remove `if` condition and only keep the `for`. The property is being called twice.